### PR TITLE
Add `subscribe to KCSC` form to the Footer

### PR DESCRIPTION
**Pull request by SevaDer14**
PT: https://www.pivotaltracker.com/story/show/178899422
### User Story
```
As a charity
To keep users updated about us
We want them to be able to subscribe to our news letter

```
### Changes Proposed:
- adds subscribe to KCSC form to footer
### Image
image

### CODE REVIEW

File: cypress/integration/desktop/desktopVisitorCanNavigateBetweenSPAViews.feature.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe("visitor can navigate between views", () => {
       });
     });
 
-    it("is expected to display footer", () => {
+    it.only("is expected to display footer", () => {
```
Comment (lukeperrin10) [inline cypress/integration/desktop/desktopVisitorCanNavigateBetweenSPAViews.feature.js:55]: maybe remove the only now :)

File: src/components/ServiceSearch.jsx [hunk 2]
```diff
@@ -15,11 +15,11 @@ import {
   InputLabel,
   MenuItem,
   Hidden,
-  useMediaQuery
+  useMediaQuery,
```
Comment (lukeperrin10) [inline src/components/ServiceSearch.jsx:18]: Formatter conflicts?

**Review by lukeperrin10 — CHANGES_REQUESTED** [review]
Just the .only in the test file.

File: cypress/integration/desktop/desktopVisitorCanNavigateBetweenSPAViews.feature.js (hunk 1 above)
Comment (SevaDer14) [reply cypress/integration/desktop/desktopVisitorCanNavigateBetweenSPAViews.feature.js]: oops)) done

File: src/components/ServiceSearch.jsx (hunk 2 above)
Comment (SevaDer14) [reply src/components/ServiceSearch.jsx:18]: no shouldn't be. probably wasn't formatted before

**Review by lukeperrin10 — APPROVED** [review]

**Pull request merged**